ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/Constants.java

```diff
@@ -329,6 +329,14 @@ public String toString() {
     public static final String CONTRACT = "contract";
     public static final String VERSION = "version";
     public static final String TITLE = "title";
+    public static final String EMAIL = "email";
+    public static final String DESCRIPTION = "description";
+    public static final String CONTACT_NAME = "contactName";
+    public static final String CONTACT_URL = "contactURL";
+    public static final String LICENSE_NAME = "licenseName";
+
+    public static final String LICENSE_URL = "licenseURL";
+    public static final String TERMS_OF_SERVICE = "termsOfService";
```

Comment:
```suggestion
    public static final String EMAIL = "email";
    public static final String DESCRIPTION = "description";
    public static final String CONTACT_NAME = "contactName";
    public static final String CONTACT_URL = "contactURL";
    public static final String LICENSE_NAME = "licenseName";
    public static final String LICENSE_URL = "licenseURL";
    public static final String TERMS_OF_SERVICE = "termsOfService";
    
```

Reply:
They are newly introduced fields for the `OPENAPI_ANNOTATION`

Reply:
Then the space in line 337 does not make any sense right ?

Reply:
It was an extra line and already fixed there https://github.com/ballerina-platform/openapi-tools/pull/1717/commits/d1287cabce88bc4f0c18f8344f282bf9881eae55